docs/services/navigation/_index.md

```diff
@@ -554,3 +554,124 @@ obstacles = await my_nav.get_obstacles()
+
+An example of an `Orientation` reading:
+
+``` golang
+imuwit.GetOrientation{o_x: -0.58, o_y: -0.37, o:z 0.72, theta: 60.93}
+```
+
+Orientation readings are taken through the movement sensor API's [`GetOrientation()`](/components/movement-sensor/#getorientation).
+If you want to read orientation, [configure a capable movement sensor](/components/movement-sensor/#configuration) on your robot.
+Additionally, follow [these instructions](/services/frame-system/#configuration) to configure the geometries of each component of your robot within the [frame system](/services/frame-system/).
+
+### Angular Velocity
+
+The following {{< glossary_tooltip term_id="model" text="models" >}} of the [movement sensor](/components/movement-sensor/) component report angular velocity measurements:
+
+- [imu-wit](/components/movement-sensor/imu/imu-wit/)
+- [imu-vectornav](/components/movement-sensor/imu/imu-vectornav/)
+- [wheeled-odometry](/components/movement-sensor/wheeled-odometry/)
+- [gyro-mpu6050](/components/movement-sensor/mpu6050/)
+
+An example of a `AngularVelocity` reading:
+
+``` go
+imu.GetAngularVelocity{x: -47.9736, y: 142.639, z: -90.1489}
+```
+
+Angular velocity readings are taken through the movement sensor API's [`GetAngularVelocity()`](/components/movement-sensor/#getangularvelocity).
+If you want to get an angular velocity reading, first [configure a capable movement sensor](/components/movement-sensor/#configuration) on your robot.
+Use angular velocity readings to determine the speed and direction at which your robot is rotating.
+
+### Position
+
+The following {{< glossary_tooltip term_id="model" text="models" >}} of the [movement sensor](/components/movement-sensor/) component report Position measurements:
+
+- [gps-nmea](/components/movement-sensor/gps/gps-nmea/)
+- [gps-nmea-rtk-pmtk](/components/movement-sensor/gps/gps-nmea-rtk-pmtk/)
+- [gps-nmea-rtk-serial](/components/movement-sensor/gps/gps-nmea-rtk-serial/)
+- [wheeled-odometry](/components/movement-sensor/wheeled-odometry/) (provides a relative estimate only based on where the base component has started)
+
+A `Position` reading specifies the GPS coordinates of an object in 3D space or its position in the geographic coordinate system [(GCS)](https://en.wikipedia.org/wiki/Geographic_coordinate_system).
+
+An example of a `Position` reading:
+
+``` go
+gps.GetPosition{x: 4.5, y: 5.6, z: 6.7}
+```
+
+Position readings are taken through the movement sensor API's [`GetPosition()`](/components/movement-sensor/#getposition).
+If you want to get a position, first [configure a capable movement sensor](/components/movement-sensor/#configuration) on your robot.
```

Comment:
```suggestion
To get a position, first [configure a capable movement sensor](/components/movement-sensor/#configuration) on your robot.
Then use the [`GetPosition()`](/components/movement-sensor/#getposition) method on the movement sensor.
```
More active, and also I think this order makes more sense since first they configure, then they use the API

Reply:
will change order, good suggestion!! Done